### ISSUE TITLE
Configset directory missing when using host-mounted directories

### ISSUE THREAD

**Issue by xcellenceit** [docker-solr/docker-solr#334]
Using latest Docker image for Solr. I have created configsets using dockerfile copying my configset files to /var/solr/data/configsets/

When I start it using named volume the, configsets works fine. 

But when I do it using host-mounted directories on windows, the configsets are missing and I don't find configset folder at this location 
/var/solr/data/

This gives me following error:

"Error CREATEing SolrCore 'my_core': Unable to create core [my_core] Caused by: Could not load configuration from directory /var/solr/data/configsets/my_core_config"

**Comment by makuk66** [comment]
Right. Don't put it inside `/var/solr/data`. Copy your configsets dir to `/opt/solr/server/solr/myconfigsets`, or just add the individual configsets to  `/opt/solr/server/solr/configsets`

**Comment by dsmiley** [comment]
FYI: https://issues.apache.org/jira/browse/SOLR-7323

**Comment by xcellenceit** [comment]
Solr Docker guide suggests to use /var/solr/ for this! 

However, I will try with this /opt/solr/server/solr/configsets

**Comment by makuk66** [comment]
> Solr Docker guide suggests to use /var/solr/ for this!

Where?

**Comment by xcellenceit** [comment]
> 
> 
> > Solr Docker guide suggests to use /var/solr/ for this!
> 
> Where?

Not sure! But I read it somewhere!! Will share if I found it again. Thanks

I am really impressed with level of commitment you guys are having to support & maintain the doc. Thank you for this. :)

**Comment by xcellenceit** [comment]
> 
> 
> Right. Don't put it inside `/var/solr/data`. Copy your configsets dir to `/opt/solr/server/solr/myconfigsets`, or just add the individual configsets to `/opt/solr/server/solr/configsets`

Hi,

This doesn't work. Check https://prnt.sc/txs2z1

Error CREATEing SolrCore 'my_core': Unable to create core [my_core] Caused by: Could not load configuration from directory /var/solr/data/configsets/my_core_config

I have my config at configsets dir to /opt/solr/server/solr/myconfigsets. 

I am trying with named volume, not sure f that makes the difference.

**Comment by makuk66** [comment]
> Could not load configuration from directory /var/solr/data/configsets/my_core_config

Seems to me you didn't specify it properly.

Follow https://github.com/docker-solr/docker-solr-examples/tree/master/custom-configset; I have confirmed just now that still works with the latest
